ci/Jenkinsfile.release

```diff
@@ -71,13 +71,20 @@ pipeline {
           variable: 'DISCORD_WEBHOOK'
         ),
       ]) {
+        def repo = [
+          url: 'https://github.com/status-im/nwaku',
+          branch: GIT_BRANCH.minus("origin/"),
+          commit: GIT_COMMIT.take(8),
+          prev: GIT_PREVIOUS_SUCCESSFUL_COMMIT.take(8),
+        ]
         discordSend(
           title: "${env.JOB_NAME}#${env.BUILD_NUMBER}",
           description: """
             Nim-Waku deployment successful!
-            Image: [`${IMAGE_NAME}:${IMAGE_TAG}`](https://hub.docker.com/r/${IMAGE_NAME}/tags?page=1&name=${IMAGE_TAG})
-            Branch: [`${GIT_BRANCH.minus("origin/")}`](https://github.com/status-im/nwaku/commits/${GIT_BRANCH.minus("origin/")})
-            Commit: [`${GIT_COMMIT.take(8)}`](https://github.com/status-im/nwaku/commit/${GIT_COMMIT.take(8)})
+            Image: [`${IMAGE_NAME}:${IMAGE_TAG}`](https://hub.docker.com/r/${IMAGE_NAME}/tags?name=${IMAGE_TAG})
+            Branch: [`${repo.branch}`](${repo.url}/commits/${repo.branch})
+            Commit: [`${repo.commit}`](${repoUrl}/commit/${repo.commit})
+            Previous: [`${repo.prev}`](${repo.url}/compare/${repo.prev}...${repo.commit})
```

Comment:
Non-blocking: `Previous` > `Comparison`

Reply:
`[`${repo.prev}`]` > [`${repo.prev}...${repo.commit}`]

Reply:
I don't think `Comparison` is a good word, since it doesn't actually indicate what we're comparing with.

Reply:
If written as `Comparison: 75f4c67...6df2317` I'd say so, but no biggie 🙂. Feel free this close this thread 👍 .

Reply:
Other keyword: Diff, Changes